mathml/presentation-markup/tokens/tokens-rendering-from-in-flow.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>tokens rendering from in-flow children</title>
+    <link rel="help" href="https://w3c.github.io/mathml-core/#token-elements">
+    <meta name="assert" content="Verify rendering of tokens is only affected by in-flow children.">
+    <style>
+      .oof {
+          left: 0;
+          position: absolute;
+      }
```

Comment:
any reason why you are not doing

```
.oof1 {
          position: absolute;
      }
      .oof2 {
          position: fixed;
      }
      .nobox {
          display: none;
      }
```
for this  one (note: the example I gave you by chat was just an example).

Reply:
Right, changed